lib/ansible/modules/messaging/rabbitmq_user.py

```diff
@@ -31,8 +31,8 @@
   password:
     description:
       - Password of user to add.
-      - To change the password of an existing user, you must also specify
-        C(force=yes).
+      - To change the password of an existing user for RabbitMQ < 3.6.5,
+        you must also specify C(force=yes).
```

Comment:
@resmo Yep. :-)
What do you think, if we write this:
> - To change the password of an existing user, when you use 
      ansible < 2.4 or RabbitMQ < 3.6.5, you must also specify C(force=yes).